### ISSUE TITLE
blank line render broken

### ISSUE THREAD

**Issue by trewjames** [trewjames/sourcetype#25]
codeBlock:
```js
    const str = `
def func:
  if foo:
    return True


func()
`;
```

wordList:
``` js
[
  ["def", "func:"],
  [tab, "if", "foo:"],
  [tab, tab, "return", "True"],
  [""],
  [""],
  ["func()"],
];
```

view:
![image](https://user-images.githubusercontent.com/66286082/130332822-c722acf7-ccda-4480-b7e4-e9c8f84493b7.png)

Probably needs a simple adjustment at `<Word/>`

**Comment by mineugene** [comment]
If we know the line height, we can apply a `height` style to all lines under CodeWrapper. But the CSS, in general, needs to be updated.

**Comment by mineugene** [comment]
Something like this in `index.css` would define a consistent line-height:
```
+#codeWrapper > .WordList {
+  height: 2.5rem;
+}
```
Note: this patch is referring to the point after the latest style update, commit 42aa421.

**Comment by trewjames** [comment]
yeah I've already made some random tweaks for #12 in terms of the cursor.

**Comment by trewjames** [comment]
btw #12 is pretty hard

**Comment by trewjames** [comment]
maybe just me but without the div outlines the spacing between lines look bit too big
![image](https://user-images.githubusercontent.com/66286082/132264855-4d891f71-00dd-4b90-a8f0-46b3a4193770.png)
